addons/hr_holidays/static/src/js/time_off_calendar.js

```diff
@@ -185,7 +185,29 @@ odoo.define('hr_holidays.dashboard.view_custo', function(require) {
                     // we reload as record can be created or modified (sent, unpublished, ...)
                     self.reload();
                 });
+                for(let i = 0; i < this.previousOpen.buttons.length; i++) {
+                    this.previousOpen.buttons[i].classes += ' timeoff_button';
+                }
+            }
+        },
+
+        _onOpenEvent: function(event) {
+            if (!this.eventOpenPopup) {
```

Comment:
evenOpenPopup is set via the params of the CalendarController. Not used in time_off_calendar but at least _onOpenEvent is kept close to what it was in CalendarController